mathics/builtin/arithmetic.py

```diff
@@ -1861,6 +1861,23 @@ class Factorial(PostfixOperator, _MPMathFunction):
     mpmath_name = "factorial"
 
 
+class Factorial2(PostfixOperator, _MPMathFunction):
+    """
+    <dl>
+    <dt>'Factorial2[$n$]'
+    <dt>'$n$!!'
+        <dd>computes the double factorial of $n$.
+    </dl>
+
+    >> 5!!
+     = 15
+    """
+
+    operator = "!!"
+    precedence = 610
```

Comment:
I can't find mention of the precedence either. 

This hasn't been added to mathics-scanner, so it would be nice to add this into the table there. 

The sympy equivalent is also `factorial2` so that should be listed as well.

Reply:
Ok.